public-ui/src/app/views/quick-search/quick-search.component.css

```diff
@@ -12,15 +12,15 @@
 
 /* result card */
 .white-card-box.search-result {
-    height: 240px;
-    width: 240px;
+    height: 260px;
```

Comment:
So for sizing I've been trying to use multiples of 24px. Clarity recommends that for the grid as that is what all elements are based on. See here: https://vmware.github.io/clarity/documentation/v0.12/app-layout

  So 264px may be better and 246px may be better for width.

Reply:
Cool thanks. That was helpful. I had to change this to fit in survey description.